io.github.cosmicUtils.cosmicAppletMinimon.json

```diff
@@ -0,0 +1,45 @@
+{
+  "id": "io.github.cosmicUtils.cosmicAppletMinimon",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "cosmic-applet-minimon",
+  "finish-args": [
+    "--socket=wayland",
+    "--filesystem=xdg-config/cosmic:ro",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=/sys/class/hwmon:ro",
+    "--filesystem=/sys/class/drm:ro",
+    "--filesystem=/sys/devices:ro",
+    "--device=all"
```

Comment:
why does it need device=all?

Reply:
I believe `device=all` is for reading some hardware information, probably for GPU usage. I think we tested and it didn't work without it. I can do another test to confirm if it is a blocker?